app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SessionsController < ApplicationController
+  def create
+    user = User.from_omniauth(env['omniauth.auth'])
+    session[:user_id] = user.id
```

Comment:
what happens if oauth doesn't work and we don't get the user back correctly. with the code you have here, things would break. we want some type of conditional to ensure that our code doesn't break. if we find the user, we set the session and redirect. if we don't find the user, what happens?